libs/blog/articles/feature-article/src/lib/article-share-icons/article-share-icons.component.ts

```diff
@@ -31,6 +48,20 @@ type ShareItem = {
           <fast-svg class="text-al-foreground" [name]="item.icon" size="28" />
         </a>
       }
+      <a
+        #copyUrlIcon
+        role="button"
+        [attr.aria-label]="t('articleShareIcons.urlAriaLabel')"
+        [cdkCopyToClipboard]="window.location.href"
+        (click)="onCopyUrl()"
+        target="_blank"
+      >
+        @if (!isCopyUrlClicked()) {
+          <fast-svg name="link" class="text-al-foreground" size="28" />
+        } @else {
+          <fast-svg name="circle-check" class="text-al-foreground" size="28" />
```

Comment:
there will be a frame where unloaded icons shows up on the screen as there are some optimizations in this component, so it's better to load them both and use classes to hide one or another